src/gennylib/include/gennylib/GlobalRateLimiter.hpp

```diff
@@ -237,6 +240,27 @@ class BaseGlobalRateLimiter {
 
 using GlobalRateLimiter = BaseGlobalRateLimiter<std::chrono::steady_clock>;
 
-}  // namespace genny::v1
+inline void simpleLimitRate(GlobalRateLimiter& rateLimiter) {
```

Comment:
This is a helper function to assist with the general case of using the rate limiter.

I figured that if it can be implemented without access to things `private` or `protected` to `GlobalRateLimiter`, then it ought to be a free function. But if you'd prefer this as a member, I can do that.

Reply:
Slight preference to make it a member simply because it makes it more discoverable using IDEs and the like.